Dockerfile

```diff
@@ -17,21 +17,19 @@
 # under the License.
 #
 
-ARG GO_VERSION=golang:1.15
-FROM apachepulsar/pulsar:2.8.2 as pulsar
-FROM $GO_VERSION as go
+ARG PULSAR_IMAGE=apachepulsar/pulsar:latest
+ARG GOLANG_IMAGE=golang:latest
```

Comment:
The version is fixed in the Makefile, so CI builds would always use the version set there.  I set these to default to latest so that they don't become out of date.  I can add a comment to the Dockerfile saying that these versions should be set via the Makefile or CLI.

Reply:
Added a comment about setting the version during build.